.github/ISSUE_TEMPLATE/00-bug-issue.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+labels: 'type:bug'
+
+---
+
+Please go to Stack Overflow for help and support:
```

Comment:
Should we also advertise the TF forum? Should we *only* advertise the TF forum? What's our policy?

Reply:
For TF repositories, we are recommending TF forum only (no stackoverflow). Supporting TF Forums is easy and manageable. Please let me know whether to update it with TF Forum link.